frontend/lib/src/components/widgets/DataFrame/hooks/useCustomTheme.ts

```diff
@@ -90,15 +105,23 @@ function useCustomTheme(): CustomThemeReturn {
       // resizeIndicatorColor?: string;
       // headerBottomBorderColor?: string;
     }
+
+    return {
+      glideTheme,
+      tableBorderRadius: theme.radii.default,
+      tableBorderWidth: 1,
+      // glide-data-grid can only handle integer pixel values:
+      defaultTableHeight: Math.round(convertRemToPx("25rem")),
+      minColumnWidth: Math.round(convertRemToPx("3.125rem")),
+      maxColumnWidth: Math.round(convertRemToPx("62.5rem")),
+      maxColumnAutoWidth: Math.round(convertRemToPx("31.25rem")),
+      defaultRowHeight: Math.round(convertRemToPx("2.1875rem")),
+      defaultHeaderHeight: Math.round(convertRemToPx("2.1875rem")),
```

Comment:
These values will likely be cleaned up in a follow-up PR. The goal in this PR was mainly refactoring it without triggering visual changes.